lib/Serialization/ModuleFormat.h

```diff
@@ -481,8 +481,10 @@ using ValueOwnershipField = BCFixed<2>;
 enum class DefaultArgumentKind : uint8_t {
   None = 0,
   Normal,
-  File,
+  FileID,
   FilePath,
+  FileIDSpelledAsFile,
+  FilePathSpelledAsFile,
```

Comment:
Serialization usually defines its own copies of enums like this so that it's more obvious that you'll need to bump SWIFTMODULE_VERSION_MINOR.

I suppose I can change the order.

Reply:
Ah, I see. Thank you for the explanation :)

Reply:
Ordering change is in the latest force push (along with another bump of SWIFTMODULE_VERSION_MINOR).

Reply:
Hm? I don't see the aligned ordering in the diff.

Reply:
The order was:
```
  FileID,
  FilePath,
  FileIDSpelledAsFile,
  FilePathSpelledAsFile,
```

It's now:
```
   FileID,
   FileIDSpelledAsFile,
   FilePath,
   FilePathSpelledAsFile,
```

There's no commit showing this because I also had to rebase the pull request when ModuleFormat.h changed out from under me, but it has been corrected.

Reply:
Ahh, sorry. I meant this whole enum. For example `Inherited` should be after `Normal`, `File` et al should be after `StoredProperty`. That's is not super important, but this is a good opportunity to do it.